jfry/next-game.txt

```diff
@@ -0,0 +1,7 @@
+Seven Wonders
+Coup
```

Comment:
I think this should move down one on the list.

<reasons>

Reply:
changed in latest commit.